crates/api/src/comment.rs

```diff
@@ -32,6 +32,7 @@ use crate::Perform;
 impl Perform for MarkCommentAsRead {
   type Response = CommentResponse;
 
+  #[tracing::instrument(skip(self, context, _websocket_id))]
```

Comment:
Here it might be worth including `self` in the trace, as it contains the params. Though at least for GET requests, these should also be included by the HTTP logger, not sure about POST.